.vib/sonarqube/goss/vars.yaml

```diff
@@ -0,0 +1,15 @@
+binaries:
+  - gosu
+  - java
+  - psql
+directories:
```

Comment:
you are missing the existence and permissions of `/opt/bitnami/sonarqube/conf/sonar.properties`

Reply:
Good catch! It seems the comment related to the permissions change of that file is wrong and I checked another one. Thanks!